server/src/main/java/io/crate/execution/engine/indexing/ShardedRequests.java

```diff
@@ -41,7 +42,7 @@ public final class ShardedRequests<TReq extends ShardRequest<TReq, TItem>, TItem
     final Map<String, List<ReadFailureAndLineNumber>> itemsWithFailureBySourceUri = new HashMap<>();
     final Map<String, String> sourceUrisWithFailure = new HashMap<>();
     final List<RowSourceInfo> rowSourceInfos = new ArrayList<>();
-    final Map<ShardLocation, TReq> itemsByShard = new HashMap<>();
+    final Map<ShardLocation, TReq> itemsByShard = new LinkedHashMap<>();
```

Comment:
What's the reason for this change? Increases memory use

Reply:
Not sure anymore why I did this, seems not needed, I'll revert. Thanks for catching!